### ISSUE TITLE
[New build - DEFAULT] 20w23c: 2mx2m Docking Port.

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#2315]
In this build I give you a second option for the docking port to be compatible with 2m wide opening.

New Features:
- Another Docking port, made to fit in a 2mx2m opening on the side of a ship.

Bugfixes
#2260 Weapon visible on reload when equipped.

**Comment by tsunamayo** [comment]
Build is live! Thanks a lot! It has been a rough few days for station but now it is finally going somewhere!

**Comment by Sullos** [comment]
Hey tsuna, I think there may have been a problem. I can't seem to find the 2x2 door. I can see the original 3x3 connector that has the round ring and the 3x3 box at the back. And then a connector piece which only is the ring.
However, the ring is still roughly 2.5x2.5m.

Also is there meant to be a visual aid for setting the bounds for the shipyard block. It's difficult to get it to line up to landing pads at the moment.

**Comment by tsunamayo** [comment]
yes so this docking port has the same size as the other one, the difference is that there is no border, only the the ring, so you can make your own border essentially. It fits in a 2x2 hole, you have to place it on the side.
For the shipyard there should be an help visible when you equip a shipyard - this is crappy I will change that.
Thanks!

**Comment by Womble-TC** [comment]
I like it! Just to clarify though, are you still wanting us to make station modules? Or has that been scrubbed along with the 4m grid stations.

**Comment by Dwarf-LordPangolin** [comment]
Thank you!! It works great, and between the 3m and 2m docking block pretty much every type of ship should have something available that will work.

I've added the dock to my modules, and I'm getting started making the landing pad set up with the shipyard block, and then am going to make a habitat module and start everything out with a proper interior.

**Comment by tsunamayo** [comment]
amazing, yes @Womble-TC I still need modules, just a tad smaller to not make the game lag. I have started to generate them proceduraly.

**Comment by itsKeq** [comment]
have a few modules just hallways for now
![20200604151403_1](https://user-images.githubusercontent.com/65949046/83717519-4314a680-a676-11ea-9477-e5da2ef77d57.jpg)
![20200604150559_1](https://user-images.githubusercontent.com/65949046/83717525-46a82d80-a676-11ea-99f8-f2273130e22d.jpg)
[30m-hallway sesctions.zip](https://github.com/tsunamayo/Starship-EVO/files/4727660/30m-hallway.sesctions.zip)

**Comment by Dwarf-LordPangolin** [comment]
Wanted to share the WIP screenshots of the small living quarters module I'm working on. It takes the 2-way hallway module as a starting point, then expands on it. The exterior is still heavily under construction, so please forgive the mess.

![20200604031444_1](https://user-images.githubusercontent.com/33041786/83732423-9f91aa80-a611-11ea-9edb-584520e34662.jpg)
![20200604031140_1](https://user-images.githubusercontent.com/33041786/83732391-97396f80-a611-11ea-8ba6-ab99c889e1c9.jpg)
![20200604031151_1](https://user-images.githubusercontent.com/33041786/83732395-986a9c80-a611-11ea-85d1-390a35093d12.jpg)
![20200604031200_1](https://user-images.githubusercontent.com/33041786/83732398-99033300-a611-11ea-8d46-2e86bda9aa2c.jpg)
![20200604031208_1](https://user-images.githubusercontent.com/33041786/83732406-9b658d00-a611-11ea-881c-0a979b71afd4.jpg)
![20200604031219_1](https://user-images.githubusercontent.com/33041786/83732412-9c96ba00-a611-11ea-86cb-b181eaeb50c1.jpg)
![20200604031226_1](https://user-images.githubusercontent.com/33041786/83732416-9dc7e700-a611-11ea-9c61-14846fbcd7ff.jpg)
![20200604031233_1](https://user-images.githubusercontent.com/33041786/83732420-9e607d80-a611-11ea-8e71-3d480fe61bea.jpg)

**Comment by Garrett-C** [comment]
Would it be possible to make it so that the docking ports can be placed based on the brick grid? It might be nice for those medium-scale ships that use bricks for the shape.
